SCClassLibrary/Common/Audio/Line.sc

```diff
@@ -26,18 +26,13 @@ LinExp : PureUGen {
 	}
 }
 
-LinLin {
+LinLin : PureUGen {
+	checkInputs { ^this.checkSameRateAsFirstInput }
```

Comment:
This may change behaviour: so far we could write `LinLin.kr(SinOsc.ar(440), -1, 1, 3, 4)`.